src/main/groovy/com/instabot/operators/weboperations/userextractor/RelatedUsersUpdater.groovy

```diff
@@ -1,5 +1,8 @@
+// TODO: long package name
```

Comment:
Thanks, I will change it to 
package com.instabot.weboperations.userextractor

I would probably not have any other operators than the web ones.